frame/contracts/src/wasm/runtime.rs

```diff
@@ -190,6 +190,8 @@ pub enum RuntimeCosts {
 	Transfer,
 	/// Weight of calling `seal_call` for the given input size.
 	CallBase(u32),
+	/// Weight of calling `seal_call_code` for the given input size.
+	CallCode(u32),
```

Comment:
What do you think about `CallCodeBase` naming?=)

Reply:
As we don't have separate `CallCodeCopyOut` (would be identical to `CallCopyOut`) doesn't seem to me like there is a reason for having 'Base" in the name